TerminalDocs/customize-settings/profile-settings.md

```diff
@@ -528,9 +532,9 @@ This sets how the profile reacts to termination or failure to launch. `"graceful
 
 ___
 
-## Bell settings
+## Bell settings ([Preview](https://aka.ms/terminal-preview))
 
-Controls what happens when the application emits a BEL character. When set to `"audible"`, the Terminal will play a sound. When set to `"none"`, nothing will happen.
+Controls what happens when the application emits a BEL character. When set to `"audible"`, the terminal will play a sound. When set to `"none"`, nothing will happen.
```

Comment:
```suggestion
Controls what happens when the application emits a BEL character. When set to `"audible"`, the Terminal will play a sound. When set to `"none"`, nothing will happen.
```
I thought we specifically want to use cap T here? Idk. We might be inconsistent with this :/

Reply:
The docs should all have lowercase t for this, unless we missed a few from community contributions.